mimic/model/clb_objects.py

```diff
@@ -66,7 +66,8 @@ class Node(object):
                  default=attr.Factory(lambda: randrange(999999)))
     status = attr.ib(validator=one_of_validator("ONLINE", "OFFLINE"),
                      default="ONLINE")
-    feed_events = attr.ib(default=[])
+    # List of (xml text, updated seconds since epoch) tuple
+    feed_events = attr.ib(default=attr.Factory(lambda: []))
```

Comment:
You can just do `Factory(list)` instead of writing your own `lambda` - a few fewer characters :).